src/Middleware/RateLimiting/src/IRateLimiterPolicy.cs

```diff
@@ -14,11 +14,11 @@ public interface IRateLimiterPolicy<TPartitionKey>
     /// <summary>
     /// Gets the <see cref="Func{OnRejectedContext, CancellationToken, ValueTask}"/> that handles requests rejected by this middleware.
     /// </summary>
-    public Func<OnRejectedContext, CancellationToken, ValueTask>? OnRejected { get; }
+    Func<OnRejectedContext, CancellationToken, ValueTask>? OnRejected { get; }
```

Comment:
Why isn't this a method?

Reply:
It matches the way we set `OnRejected` on the options - but, given that we made `GetPartition` a method in the interface vs a `Func` on the options, it'd probably be reasonable to do the same in the next iteration - I can bring it up at the next API proposal

Reply:
The reason it's a `Func` instead of a method is because of the `policy.OnRejected is not null` check [here](https://github.com/dotnet/aspnetcore/pull/42417/files#diff-7a9373c0644d34a53761c205c33d884646e86e13a06575f0c2f013f26a8128efR82). This seems preferrable than adding a `bool OverridesOnRejected { get; }` to determine whether the policy.

Maybe it's okay if we always run the global `OnRejected` `Func` and the policy's `OnRejected` method though.